ON_CALL.md

```diff
@@ -1,48 +1,61 @@
  
-## What you need before going on call
+## Before you go on call for the first time, make sure you have: 
 
+* Can't figure out how to get access? Ask Jamie Winsor.
+* You should be a member of the `core` origin and habitat-sh github group.
 * Access to the habitat team [1Password Shared vault](https://team-habitat.1password.com)
 * Access to [PagerDuty](https://opscode.pagerduty.com) - ask #helpdesk if you need an account
 * Access to [Datadog](https://app.datadoghq.com)
 * Access to [Pingdom](https://my.pingdom.com) (creds in Chef LastPass) - Pingdom currenty monitors - https://willem.habitat.sh/v1/depot/origins/core
-* Ability to ssh to our production environment (see below)
-* Basic familiarity with the services we run and how they are running (detailed below)
+* Access to [statuspage.io](https://statuspage.io) - Jamie Winsor owns admin
+* Ask #helpdesk to add the Habitat AWS Portal app to your OKTA dashboard.
+* Ability to ssh to our production environment
+* Basic familiarity with the services we run and how they are running
  
 ## Responsibilities
  
 * Available to respond to PagerDuty alerts (if you are going to be away from a computer for an extended period, you are responsible for getting someone to take on call).
-* Incident response does not mean you need to solve the problem yourself.
-* You are expected to diagnose, to the best of your ability, and bring in help if needed.
-* Communication while troubleshooting is important.
-* Triage incoming GitHub issues and PRs (best effort - at least tag issues if possible and bring critical issues to the forefront).
-* Monitor #general and #operations as time permits.
+* If you are on secondary on-call and receive a page, you are responsible for making sure your help is not needed. 
+* You are expected to diagnose, to the best of your ability, and bring in help if needed to resolve.
+
+## Communication Responsibilities
+* Declare an operational issue on statuspage.io. This will automatically post an alert to habitat's slack #general channel, update our statuspage, and tweet.
+* You are responsible for informing your team that you are taking charge of the issue, and you are responsible for telling your team that you are unable to continue responding to that issue before you stop working on it.
+* You are not responsible for responding to questions or community questions. 
+* You are responsible for updating statuspage.io as status changes. 
+* You are not responsible for posting minutes during the issue if status has not materially changed. All updates go into statuspage.io, no other venue.
+* Tone in statuspage.io communication should be friendly and professional.
+* You are responsible for escalating immediately to someone more senior or knowledgeable if you cannot fix the issue alone or need help. Never be afraid to start phoning people, regardless of time or day. Getting production into a good state should be your only concern.
```

Comment:
Do we have people's contact info somewhere?  I can't phone anyone on our team right now if I needed to.

Reply:
@raskchanky let's start a spreadsheet